gammapy/background/tests/test_models.py

```diff
@@ -43,3 +50,95 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+def test_wcs_object():
+    bins_x = Angle([1., 2., 3., 4.], 'degree')
+    bins_y = Angle([-1.5, 0., 1.5], 'degree')
+    wcs = make_linear_wcs_from_bin_edges_arrays("X", "Y", bins_x, bins_y)
+    nbins_x = len(bins_x) - 1
+    nbins_y = len(bins_y) - 1
+    reco_bins_x, reco_bins_y = make_linear_bin_edges_arrays_from_wcs(wcs,
+                                                                     nbins_x,
+                                                                     nbins_y)
+
+    # test: reconstructed bins should match original bins
+    assert_allclose(reco_bins_x, bins_x)
+    assert_allclose(reco_bins_y, bins_y)
+
+
+class TestCubeBackgroundModel():
+
+    @remote_data
+    def test_read_bin_table(self):
+
+        # test shape of bg cube when reading a file
+        filename = '../test_datasets/background/bg_cube_model_test.fits'
+        filename = datasets.get_path(filename, location='remote')
+        bg_cube_file = CubeBackgroundModel.read_bin_table(filename)
```

Comment:
OK, I renamed the object to `bg_cube_model`.

For the `read` method: it is no problem to do it. But if we want to be consistent, the write method should do something similar. The problem is that the `write` methods are defined as:
```python
def write_bin_table(self, *args, **kwargs):
def write_image(self, *args, **kwargs):
```
So it is hard to pass the format argument. I tried the following, but It didn't work:
```python
def write(self, format='bin_table', *args, **kwargs):
```
@cdeil Do you have any suggestion/example?